control-plane/roles/isolated-clusters/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+isolated_clusters_ntp_image_name: ghcr.io/mwennrich/chrony
+isolated_clusters_ntp_image_tag:
+isolated_clusters_ntp_namespace: ntp-service
+
+isolated_clusters_dns_image_name: coredns/coredns
+isolated_clusters_dns_image_tag:
+isolated_clusters_dns_namespace: dns-service
+
+isolated_clusters_ingress_ingress_controller_namespace: ingress
+isolated_clusters_ingress_ingress_controller_chart_version:
+isolated_clusters_ingress_ingress_controller_chroot: true
+isolated_clusters_ingress_cert_manager_version:
+isolated_clusters_ingress_cert_manager_cluster_issuer: letsencrypt-gcp-dns
```

Comment:
Pre-suggesting the DNS provider is maybe something quite opinionated. Still not having a good idea yet but I think it would be good to give a bigger freedom to how users want to deal with where certs and ingresses come from. I will come back to this.

Reply:
Drop cert manager (part of actual deployment of user)